### ISSUE TITLE
[EN][Blade's Edge Mountains][Object] Apexis Shards

### ISSUE THREAD

**Issue by aendry** [Atlantiss/BugTracker#9180]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: [the object](http://www.wowhead.com/object=185911/apexis-shard-formation) don't have the shining sparks

**Current behaviour**: What happens is that you can't see the diference betwen a shard and a non shard

**Expected behaviour**: to spark and shine and gliter?

**Steps to reproduce the problem**:

1. just check the area
2. 
3. 

**Revision**: 5417

**Comment by Piklo** [comment]
add wowhead links

**Comment by aendry** [comment]
can be closed. I just read this comment

> 

By Daindurin (3,124 – 4·25) on 2012/01/09 (Patch 4.3.0)
**Sadly, these have been overlooked by many. They are difficult to spot because they blend in with the surrounding crystal formations: no sparkles, no give away outlines or color variation. Every crystal mound in the west plateau has the potential to spawn one and only one of these formations, (edit: but not every mound at once, random like the eggs on Netherwing Ledge but much tighter area). If you pass your mouse over the mound area your mouse will turn to a "working gear" and the crystal will briefly highlight. Fly too fast and you'll miss it. I canvased the area for about an hour. The re-spawn rate is good it seems as I always found one within a minute or two. Of course this is not factoring in that a flayer is usually nearby and will attack. Result after one hour: 32. Add in the 10 I got from the friendly flayers nearby, not bad. So if you get tired of killing things ('cept flayers), try mining these (and no you do not have to be a miner). Happy hunting.**
_Last edited by Daindurin on 2012/01/09 (Patch 4.3.0)_
